LES.csproj

```diff
@@ -24,22 +26,31 @@
     <PackageReference Include="BepInEx.PluginInfoProps" Version="1.*" />
     <PackageReference Include="UnityEngine.Modules" Version="2020.3.16" IncludeAssets="compile" />
   </ItemGroup>
+
+<PropertyGroup Condition=" '$(USERNAME)' == '' ">
```

Comment:
I added my system's username here, but it wouldn't work. These hard coded paths are a mess... Do you care to just change it back for your next change?